tests/testthat/test-a11y_panel.R

```diff
@@ -0,0 +1,152 @@
+test_that("Accessibility dates are required", {
+  expect_no_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2024",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    )
+  )
+  expect_no_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26 November 2024",
+      date_prepared = "28/11/2024",
+      date_reviewed = "12-03-2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    )
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_tested\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26/11/24",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_prepared\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2024",
+      date_prepared = "28th November 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_reviewed\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2025",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "26th November 2025 is in the future."
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "Geoff",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "Geoff not in a valid date format."
```

Comment:
I enjoy the example, though makes me wonder if the error message should actually be `date_tested is not in a valid format` so it's clear the argument is wrong as that's the bit to reference in the docs if there's an issue? 

A more informative error message again could be to include an example of the format, `date_tested is not in a recognise date format, e.g. 8th August 2024 or 08/08/2024`

Reply:
Done

Reply:
Almost, you've typo'd 'Auguest' in the error message

Reply:
Fixed